src/java/org/apache/cassandra/service/accord/db/AccordUpdate.java

```diff
@@ -138,7 +138,7 @@ public Type type()
 
         public boolean applies(FilteredPartition partition)
         {
-            return applies(partition.getRow(clustering));
+            return applies(partition == null ? null : partition.getRow(clustering));
```

Comment:
this was a bug found while running the `multipleShards` test; can't pull a row from a null partition to check if row exists.